src/CribblyBackend.DataAccess/Repositories/GameRepository.cs

```diff
@@ -47,6 +48,32 @@ public async Task<Game> GetById(int id)
             game.Teams = teams.Values.ToList();
             return game;
         }
+        public async Task<List<Game>> GetByTeamId(int id)
+        {
+            var games = (await connection.QueryAsync<Game, Team, Team, Game>(
+                @"
+                    SELECT * FROM Scores s 
+                    LEFT JOIN Games g on s.GameId = g.Id 
+                    LEFT JOIN Teams t on s.TeamId = t.Id 
+                    LEFT JOIN Scores s2 on s.GameId = g.Id
```

Comment:
Would it be clearer to say `LEFT JOIN Scores s2 on s.GameId = s2.GameId`? This would also eliminate the need to do the `AND` on L61